x-pack/plugin/security/src/main/java/org/elasticsearch/xpack/security/authc/TokenService.java

```diff
@@ -460,13 +467,14 @@ private static void decryptTokenId(StreamInput in, Cipher cipher, Version versio
     public void invalidateAccessToken(String tokenString, ActionListener<Boolean> listener) {
         ensureEnabled();
         if (Strings.isNullOrEmpty(tokenString)) {
+            logger.trace("No token-string provided");
```

Comment:
I'm not sure. Given that we already report the error via the `onFailure`, the logging _shouldn't_ be necessary at all. It's just here in case the caller isn't doing proper error handling and the ES admin wants to debug it.
To put it another way, if we were `throw`ing the exception, I probably wouldn't `WARN` here, because the caller should log the error, and we don't need the same message being logged every step up the stack trace.

If you're keen, I can change it, but my feeling is to just leave it at trace (or debug).